.github/workflows/d4l-ci-publish-release.yml

```diff
@@ -1,14 +1,14 @@
-name: D4L Publish Artifact
+name: D4L CI - Publish Release
 
 on:
   push:
     tags:
       - 'v*'
 
 jobs:
-  build:
+  publish-release:
 
-    runs-on: ubuntu-latest
+    runs-on: [self-hosted, macos]
```

Comment:
Comparing with the SDK one, I am not quite understand, why this is self hosted and the other not - could explain me the reason for this, please?

Reply:
Then we need to patch the SDK one too. The release is patching another GitHub Repository and this is only possible with a SSH key of the user that is patching. For the self-hosted machines it is already configured. The public runners need to be configured in that way but that requires some extra effort.

Reply:
Thx!